src/components/blocks/categoriesBlock.tsx

```diff
@@ -0,0 +1,22 @@
+import styles from "./blocks.module.scss";
+import GameCategory from "@/components/gameCategoryCard/gamesCategory";
+import { CategoryInfo, categoryInfos } from "@/categoriesInfo";
+
+interface BlockName {
+  blockName: string;
+}
+
+const CategoriesBlock = (blockName: BlockName) => (
+  <div className={styles.block}>
+    <div className={styles.blockHeader}>{blockName.blockName}</div>
+    <div className={styles.blockContent}>
+      {categoryInfos.map((info: CategoryInfo) => (
+        <div className={styles.blockContentElement}>
+          <GameCategory name={info.name} image={info.image} url={info.url} key={info.key} />
```

Comment:
use destructuring inside component
ex: const GameCategory = ({ name, image, url, key })

or use 1 prop data and it will be `<GameCategory info={info}/> and inside component const GameCategory = ({info})`

Reply:
the second variant you proposed is used here and in the next punct